www/html/ci/application/controllers/Landing.php

```diff
@@ -113,8 +113,8 @@ public function login() {
         $this->form_validation->set_rules('password', 'Password', "required|md5|callback_password_check",
         	array(
 				'required' => 'You must provide a %s.',
-				'password_check' => 'Password does not match Username',
-				'md5' => 'didnt work'
+				'password_check' => 'Password does not match Username.',
```

Comment:
No matter what type of error happens (incorrect password or incorrect username), let's just show an error message saying, "An error occured". Let's do this so that the hackers don't know what is wrong with their credentials (so that they don't know whether the password is wrong, or the username is wrong).